### DISCUSSION TITLE
How to enable debug logs ?

### DISCUSSION THREAD

**Discussion by r0goyal** [goreleaser/goreleaser#1609] [Q&A] [answered]
Hi,

I am integrating artifactory with goreleaser. For some reason, I am getting EOF 

   ⨯ release failed after 1.94s error=artifactory: failed to publish artifacts: artifactory: upload failed: EOF

How do I enable debug logs to figure out what's wrong with my env ?

**Comment by caarlos0** [comment, marked as answer]
```
goreleaser --debug
```

also `--help` should give you all options available :)

**Comment by r0goyal** [comment]
@caarlos0 Thanks a lot
